editor/store/selectors.js

```diff
@@ -300,6 +300,44 @@ export function isEditedPostEmpty( state ) {
 	);
 }
 
+/**
+ * Returns true if the post can be autosaved, or false otherwise.
+ *
+ * @param  {Object}  state Global application state
+ * @return {boolean}       Whether the post can be autosaved
+ */
+export function isPostAutosavable( state ) {
```

Comment:
Is it "saveable" or "savable"? 

We should pick one and keep consistency. Currently this is inconsistent with `isPostSaveable`

Reply:
Its either: https://en.oxforddictionaries.com/definition/saveable

Since you already have the shorter use, I'll update my PR with this variation.

Reply:
fixed in https://github.com/WordPress/gutenberg/pull/6257/commits/bb26c8cc580658e06dc0dcfc76525b6bf2c185fc